### DISCUSSION TITLE
After last neovim update it started to highlight spaces

### DISCUSSION THREAD

**Discussion by augustocdias** [lukas-reineke/indent-blankline.nvim#368] [Q&A] [answered]
I use neovim develop and after I updated this morning I started to see blankline highlighting spaces.

![image](https://user-images.githubusercontent.com/1080789/157009597-58c202af-74ed-4f90-b7a3-ef3112d7a0e8.png)

If you look the screenshot you'll notice a lighter color in the spaces. Is it possible to disable this somehow?

This is my config:

```lua
require('indent_blankline').setup({
    char = '|',
    filetype_exclude = { 'packer', 'startup' },
    buftype_exclude = { 'terminal' },
})
```

If I disable blankline the issue disappears.

**Comment by lukas-reineke** [comment, marked as answer]
Looks like a bug in Neovim. Window positioned virtual text is completely broken as well.
I don't have time today, I'll investigate tomorrow and open an issue. For now, I suggest to either disable indent-blankline or to roll back.

**Reply by lukas-reineke** [reply]
I opened an issue here https://github.com/neovim/neovim/issues/17638

**Reply by lukas-reineke** [reply]
It is fixed in the latest version of Neovim
